gimme_aws_creds/aws.py

```diff
@@ -0,0 +1,132 @@
+"""
+Copyright 2018-present Engie SA.
+Licensed under the Apache License, Version 2.0 (the "License");
+You may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and* limitations under the License.*
+"""
+import getpass
+import re
+import sys
+import time
+import uuid
+from codecs import decode
+from urllib.parse import parse_qs
+from urllib.parse import urlparse
+from . import version
+
+import keyring
+import requests
+from bs4 import BeautifulSoup
+from keyring.backends.fail import Keyring as FailKeyring
+from keyring.errors import PasswordDeleteError
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+import base64
+from collections import namedtuple
+import xml.etree.ElementTree as ET
+
+import gimme_aws_creds.common as commondef
+
+class AwsResolver(object):
+    """
+       The Aws Client Class performes post request on AWS sign-in page
+       to fetch friendly names/alias for account and IAM roles
+    """
+
+    def __init__(self, verify_ssl_certs=True):
+        """
+        :param verify_ssl_certs: Enable/disable SSL verification
+        """
+        self._aws_signin_url = 'https://signin.aws.amazon.com/saml'
```

Comment:
Instead of hardcoding this value, you need to detect what AWS partition (AWS-China, GovCloud) the SAML response was generated for.  This is already being determined because it's needed by boto, so it just needs to get passed into this class.

Reply:
Yes you're right. I'll change this.